x-pack/plugins/observability_solution/profiling/common/storage_explorer.ts

```diff
@@ -4,16 +4,13 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import {
+  IndexLifecyclePhaseSelectOption,
+  indexLifeCyclePhaseToDataTier,
```

Comment:
nit: import `type` for `indexLifeCyclePhaseToDataTier`

Reply:
`indexLifeCyclePhaseToDataTier ` isn't a type